migrations/schemas/20221116011456-create_project_member_positions_table.sql

```diff
@@ -0,0 +1,23 @@
+
+-- +migrate Up
+CREATE TABLE IF NOT EXISTS project_member_positions (
```

Comment:
Need to drop the position column in the project_members table?

Reply:
updated